src/chaat/errors.clj

```diff
@@ -0,0 +1,20 @@
+(ns chaat.errors)
```

Comment:
suggestion: this should be renamed to error utils namespace.

An errors namespace should ideally have any custom types of errors you add

Reply:
Hmmm, understood. Are you suggesting something like this `chaat/errors/utils.clj`?